src/Pages/News.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { Spinner } from "reactstrap";
+import NewsList from "../Components/News/NewsList";
+import NoResults from "../Components/Shared/NoResults";
+import Pagination from "../Components/Shared/Pagination";
+import DotCMSApi from "../libs/dotcms.api";
+import "./News.css";
+
+class NewsPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      loading: true,
+      news: [],
+      query: `{ 
+            "query": {
+                "bool": {
+                    "must": {
+                        "query_string" : {
+                            "query" : "+contentType:news +languageId:1 +conHost:CONFHOSTVALUE"
+                        }
+                    }
+                }
+            },
+            "sort" : [
+                { "SORTBYVALUE" : {"order" : "SORTTYPEVALUE"}}
+            ],
+            "from": OFFSETVALUE,
+            "size": SIZEPERPAGE
+        }}`,
+      url: "/api/es/search"
+    };
+
+    this.pageChange = this.pageChange.bind(this);
+  }
+
+  async getNewsData(query) {
```

Comment:
There should be a method in the library that we can do like: 

```
DotCMSApi.esSearch({
    contentType: 'news',
    language: 1,
    sort: 'NOT_SURE_WHAT'
}).then(news => ...)
```

So other devs can use that to get any content they need.

The `current site` ask @freddyucv if we need it because maybe the endpoint get it in backend if we don't pass it.